sections/announcement-bar.liquid

```diff
@@ -67,6 +81,30 @@
           "type": "url",
           "id": "link",
           "label": "t:sections.announcement-bar.blocks.announcement.settings.link.label"
+        },
+        {
+          "type": "header",
+          "content": "t:sections.announcement-bar.blocks.announcement.settings.padding.block_padding_heading"
+        },
+        {
+          "type": "range",
+          "id": "padding_top",
+          "min": 0,
+          "max": 100,
+          "step": 2,
+          "unit": "px",
+          "label": "t:sections.announcement-bar.blocks.announcement.settings.padding.padding_top",
+          "default": 10
+        },
+        {
+          "type": "range",
+          "id": "padding_bottom",
+          "min": 0,
+          "max": 100,
+          "step": 2,
+          "unit": "px",
+          "label": "t:sections.announcement-bar.blocks.announcement.settings.padding.padding_bottom",
```

Comment:
I think you can go back  to what you had before here so: `t:sections.all.padding.padding_top` and `t:sections.all.padding.padding_bottom`

Reply:
Cool, thanks! Should be all set now and ready for a re-review.